homeassistant/components/sms/__init__.py

```diff
@@ -1,27 +1,40 @@
 """The sms component."""
-import voluptuous as vol
+import logging
+
+import voluptuous as vol  # pylint: disable=import-error
```

Comment:
Why is this causing an import error that needs to be disabled?

Reply:
It seems pylint does not know about the gammu lib for some reason.

Reply:
This comment is made on de Voluptuous import, not the gamma library

Reply:
Ahh I remember now. Pylint entered an infinite loop complaining about wrong import order. X before Y. 
When trying to fix it then it complain than Y before X as so on.
The only I was able to fix it was to ignore the import.
I should have added a comment.

Reply:
That sounds incorrect or maybe an issue in your environment. We don't use pylint for import ordering, as a matter of fact, the order is disabled:

![CleanShot 2022-04-27 at 19 00 14](https://user-images.githubusercontent.com/195327/165580044-3e6fae1d-fb9d-471e-b8f5-7e8715dcf5c7.png)

The error you are disabling in the comment is an `import-error`, which should not occur for voluptuous

Reply:
Perhaps you are correct.
My dev environment is based on WSL and by following the handbook I see a failure in scripts/setup complaining about a bad parameter for pip. I ask in the dev community channel but got no solutions. 

I workaround this by removing the parameter which then problaby corrupted my pylint.

I will remove all of these in my real Ubuntu dev box and see if I get better results.